### DISCUSSION TITLE
I cannot verify my student identity and the system prompts that I need to modify my Billing Information.

### DISCUSSION THREAD

**Discussion by ianchb** [community/community#84814] [GitHub Education]
### Hi everyone,

I am currently a high school student from China. I encountered difficulties when applying for student certification. Even though I filled in the name in the billing information accurately, I was still rejected.
On the materials I have, my name is written in simplified Chinese. Because GitHub does not accept single characters as valid information when filling in "last name", I tried filling in both "last name" and "first name" with my full name, or adding the English (Chinese pinyin) of my name in the remarks. But no matter what I can't pass the verification.
![Screenshot_20240107-140013_Edge](https://github.com/community/community/assets/45872450/9d8d8705-d164-4bac-8151-154e3786b3f5)

In addition, my application was approved twice before, but since I tried to verify my student status again last year, I have been rejected nearly 120 times. Although I don't have a school email, here are what I can provide: school ID, city pass for students, transcript, scholarship letter, library card, and even my health check report in school... I really have no idea how to get myself verified.
(The pictures below have been processed and the original names and other information have been erased.)
![2024-01-07-14-15-32-554~2](https://github.com/community/community/assets/45872450/1b832bcb-16ae-446f-a887-fd0954b6e1a2)
![2024-01-07-14-17-20-370~2](https://github.com/community/community/assets/45872450/d5bcf61c-7ba7-43d3-8174-a678f4aa26b6)
![2024-01-07-14-18-58-595~2](https://github.com/community/community/assets/45872450/cb0f926b-2c7b-401c-8721-6d312d5015bb)
![2024-01-07-14-20-27-781~2](https://github.com/community/community/assets/45872450/abef0eb8-f8bb-4351-9c8d-586086644e7a)

**Comment by fury-05** [comment]
I will suggest you use English Documents for verifications purpose, you can ask you college / university to provide you with letter of identity have information like when you took admission and for how long is your course ( It's also called Study Right Certificate ).

Also ask them to mention your name in English on letter / study certificate.

Additional you can use academic email for authenticity, you will need verification even after this.

**Reply by ianchb** [reply]
Thank you for your help. I am currently a high school student and my school doesn't provide email for students. Maybe I will try to re-apply when I enter university.
